spec/fixtures/factories.rb

```diff
@@ -761,7 +761,7 @@
           "digital_market_research_publisher" => %w[gambling-commission],
           "digital_market_research_area" => %w[media-and-entertainment],
           "digital_market_research_topic" => %w[future-connectivity],
-          "digital_market_research_publish_date" => "2021-02-18T10:12:26+00:00",
```

Comment:
Has the behaviour changed or was this fixture data always wrong?

Reply:
I believe it must have always been wrong, when I ran this through the 'valid_against_schema' shared example it complained about the time element. 

I don't think this was caught previously because the drcf finder didn't have it's own model spec file to check it

Reply:
Same goes for animal disease case